### PR TITLE
Add RocketBI document 

### PR DESCRIPTION
**Pull request by dnsang**
RocketBI is a dedicated BI web-based product for Clickhouse. In this PR, we add guidelines for people to install, and create their first dashboard.

### CODE REVIEW

File: docs/en/integrations/data-visualization/rocketbi-and-clickhouse.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+<img width="800" alt="github_rocketbi2" src={require('./images/rocketbi_01.gif').default}/>
+<br/>
+
+You can checkout [the Dashboard via this link.](https://demo.rocket.bi/dashboard/sales-dashboard-7?token=7eecf750-cbde-4c53-8fa8-8b905fec667e)
+
+## INSTALL
+
+Before we go deeper in create an advance dashboard with Rocket.BI, making sure you have rocket.bi platform on your ClickHouse server and the data is connected & up-to-date. Here are the step by step guide on the configuration:
+
+Follow the guide here: [Rocket.BI Readme](https://github.com/datainsider-co/rocket-bi/blob/main/README.md)
```
Comment (mshustov) [inline docs/en/integrations/data-visualization/rocketbi-and-clickhouse.md:25]: Please provide instructions on setting up a connection between ClickHouse and RocketBI. This content https://github.com/datainsider-co/rocket-bi/blob/main/README.md#get-started is the more challenging part for a newcomer.

File: i18n/en/code.json [hunk 2]
```diff
@@ -0,0 +1,280 @@
+{
```
Comment (mshustov) [inline i18n/en/code.json:1]: `i18n` folder content is not relevant to changes. Is it?

File: docs/en/integrations/data-visualization/rocketbi-and-clickhouse.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+<img width="800" alt="rocketbi_create_chart" src="https://user-images.githubusercontent.com/91059979/194799689-3ad88958-b7a8-4b3f-a98d-7af8b198d684.gif"/>
+See hi-res tutorial on Youtube: https://www.youtube.com/watch?v=TMkdMHHfvqY
+
+### Build the Chart Controls
+
+#### Create a Metrics Control
+In the Tab filter, select metric fields you want to use. Make sure to keep check on aggregation setting.
+<img width="650" alt="rocketbi_chart_6" src="https://user-images.githubusercontent.com/91059979/194493404-dd6199bc-2faf-4a73-b72e-a6370dc490f8.png"/>
+
+Rename filters & Save Control to Dashboard <img width="400" alt="Metrics Control" src="https://user-images.githubusercontent.com/91059979/194493738-c085fa53-173b-495a-b654-bffcd092b2e6.png"/>
```
[the remark this answers is not in the document]
Comment (dnsang) [reply docs/en/integrations/data-visualization/rocketbi-and-clickhouse.md]: done, please check.

File: docs/en/integrations/data-visualization/rocketbi-and-clickhouse.md (hunk 1 above)
Comment (dnsang) [reply docs/en/integrations/data-visualization/rocketbi-and-clickhouse.md]: done, please check.
Basically, I add a simple instruction for people to try it out using docker. 
`wget https://raw.githubusercontent.com/datainsider-co/rocket-bi/main/docker/docker-compose.yml `
`wget https://raw.githubusercontent.com/datainsider-co/rocket-bi/main/docker/.clickhouse.env`
Edit `.clickhouse.env`, add clickhouse server information.
`docker-compose up -d . `

**Review by mshustov — APPROVED** [review]

**Pull request merged**